addons/mail/static/src/chatter/web_portal/chatter.js

```diff
@@ -237,11 +237,14 @@ export class Chatter extends Component {
                     text
                 )}</span>`;
             });
-        const formatter = new Intl.ListFormat(user.lang?.replace("_", "-"), { type: "unit" });
         if (this.state.thread && this.state.thread.recipients.length > 5) {
-            recipients.push("…");
+            recipients.push(
+                _t("%(recipientCount)s more", {
```

Comment:
this part can be escaped too

Reply:
Thanks for the review, done :slightly_smiling_face: